vendor/vendor.json

```diff
@@ -57,6 +57,12 @@
 			"revisionTime": "2016-08-17T15:48:24Z"
```

Comment:
Okay.. had to change a bunch of other stuff to update client_golang but here is it.